templates/components/pager.html.twig

```diff
@@ -72,20 +74,26 @@
         'no_onchange': fluid_search|default(false),
         'select_class': 'search-limit-dropdown',
     }) %}
-    <span class="search-limit d-none d-md-block">
-        {{ __('%s rows / page')|format(limitdropdown)|raw }}
-    </span>
-    <span class="search-limit d-block d-md-none">
+    {% if not short_display %}
+        <span class="search-limit d-none d-md-block">
+            {{ __('%s rows / page')|format(limitdropdown)|raw }}
```

Comment:
```suggestion
            {{ __('%s rows / page')|e|format(limitdropdown)|raw }}
```
I think if you format a localization string and then output as raw, you need to escape the text just before formatting in the rare case that a translation can have HTML characters.